pkg/test-infra/abtest/operation.go

```diff
@@ -37,48 +37,45 @@ func New(cli client.Client, tidbClient *tidb.TidbOps) *Ops {
 // Apply abtest cluster
 func (t *Ops) Apply(tc *Recommendation) error {
 	var g errgroup.Group
-
 	g.Go(func() error {
-		return t.ApplyTiDBCluster(tc.Cluster1, tidb.ConfigMaps{
-			Tikv: fixture.Context.TiKVConfigMap,
-			Tidb: fixture.Context.TiDBConfigMap,
-			Pd:   fixture.Context.PDConfigMap,
+		return t.ApplyTiDBCluster(tc.Cluster1, tidb.Config{
+			Tikv: fixture.Context.TiKVConfigFile,
+			Tidb: fixture.Context.TiDBConfigFile,
+			Pd:   fixture.Context.PDConfigFile,
 		})
 	})
 	g.Go(func() error {
-		return t.ApplyTiDBCluster(tc.Cluster2, tidb.ConfigMaps{
-			Tikv: fixture.Context.ABTestConfig.TiKV2ConfigMap,
-			Tidb: fixture.Context.ABTestConfig.TiDB2ConfigMap,
-			Pd:   fixture.Context.ABTestConfig.PD2ConfigMap,
+		return t.ApplyTiDBCluster(tc.Cluster2, tidb.Config{
+			Tikv: fixture.Context.ABTestConfig.TiKVConfigFile,
+			Tidb: fixture.Context.ABTestConfig.TiDBConfigFile,
+			Pd:   fixture.Context.ABTestConfig.PDConfigFile,
```

Comment:
If b cluster's config is empty, should we use the same config file as a cluster?

Reply:
it's also reasonable for me that use the default config to start the b cluster when the b cluster's config is empty.

Reply:
LGTM